app/templates/splash.html

```diff
@@ -3,9 +3,20 @@
 {% set unauthenticated = True %}
 
 {% block content %}
-<h3>A better directory of the student body!</h3>
-<p>Yalies is a redesigned version of Yale's official <a href="https://students.yale.edu/facebook">Face Book</a>, which provides contact information for Yale students. It displays data from the Face Book and <a href="https://directory.yale.edu">Yale Directory</a>, but through a more modern, better designed, and more secure interface. Unlike official Yale sources, Yalies is accessible only to undergraduates, and censors private details that we believe Yale should not divulge. Learn how to remove your data from Yale's official sources <a href="faq">here</a>.</p>
-<p>Yalies also provides an <a href="/apidocs">API</a> for student developers. The API powers numerous student projects and university services, including <a href="https://coursetable.com">CourseTable</a>, <a href="https://yalemenus.com">Yale Menus</a>, <a href="https://rdb.yale.edu">Yale Research Database (RDB)</a>, <a href="https://yaledailynews.com/blog/2020/12/03/40-2-percent-of-returning-student-athletes-take-leaves-of-absence-this-fall">Yale Daily News</a>, <a href="https://yalies.me">Yalies.me</a>, <a href="https://mailyale.com">MailYale</a>, <a href="https://comethru.io">Comethru</a>, <a href="https://ypost.app">YPost</a>, <a href="https://github.com/ErikBoesen/yale">yale CLI</a>, <a href="https://yaledailynews.com/blog/2021/02/11/ship-lets-yalies-play-matchmaker-with-peers">Ship</a>, <a href="https://www.internships.yesatyale.org/">YES Internships</a>, </a>students in S&DS 315 and CPSC 490, and more.</p>
-<p>Yalies is maintained by the <a href="http://yalecompsociety.org">Yale Computer Society</a>.</p>
-<a class="button" href="/login/">Log in with CAS</a>
-{% endblock %}
+<div class="splash-flex">
+    <div class = "splash-text">
+        <h1>The best directory of the student body.</h1>
+        
+        <a class="button" href="/login/">Log in with CAS</a>
+
+    </div>
+        <img src="/static/images/splash.svg" class="splash-image"> 
+</div>
+
+<!-- {% endblock %} -->
+
+
+<!-- <h3>An optimized version of Yale's official <a href="https://students.yale.edu/facebook">Face Book</a>.</h2>
+<p class="splash">Yalies provides contact information for Yale students, displaying data from the Face Book and the <a href="https://directory.yale.edu">Yale Directory</a>, through a more modern and secure interface. Unlike official Yale sources, Yalies is accessible only to undergraduates, and censors private details that we believe Yale should not divulge. Learn how to remove your data from Yale's official sources <a href="faq">here</a>.</p>
```

Comment:
Yeah as long as this content gets put back as bullet points (or however you want it) I'm good w this!! Thank you so much